### PR TITLE
projects: Add Aya

### PR DESCRIPTION
**Pull request by dave-tucker**
This commit adds Aya to the list of Rust projects.

Screenshot_20210812_190707

/cc @alessandrod

### CODE REVIEW

**Review by lmb — APPROVED** [review]

**Comment by dthaler** [comment]
@dave-tucker Please rebase to resolve conflicts so we can merge this.  Thanks!

**Comment by dthaler** [comment]
@dave-tucker never mind, it was easy to manually resolve the conflict in github's UI

File: src/pages/projects.js [hunk 1]
```diff
@@ -433,7 +436,11 @@ const ProjectDescriptions = () => (
           <p>
             <a href="https://github.com/redsift/redbpf">redbpf</a> is a Rust
             eBPF toolchain that contains a collection of Rust libraries to work
-            with eBPF programs.
+            with eBPF programs.{" "}
```
Comment (dthaler) [inline src/pages/projects.js:439]:
```suggestion
            with eBPF programs.
          </p>
          <p>
```

**Comment by dthaler** [comment]
Merged per BSC meeting discussion 2021-11-10

**Pull request merged**